dags/fxa_basket_export.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime, timedelta
+
+from airflow import models
+
+# Backported 1.10.2 bqoperator
+from operators.backport.bigquery_operator_1_10_2 import BigQueryOperator
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
+from airflow.contrib.operators.gcs_delete_operator import (
+    GoogleCloudStorageDeleteOperator,
+)
+from airflow.contrib.operators.gcs_to_s3 import GoogleCloudStorageToS3Operator
+from operators.bq_sensor import BigQuerySQLSensorOperator
+
+
+default_args = {
+    "owner": "jklukas@mozilla.com",
+    "start_date": datetime(2020, 7, 21),
+    "email": ["telemetry-alerts@mozilla.com", "jklukas@mozilla.com",],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 3,
+    "retry_delay": timedelta(minutes=30),
+}
+
+GCS_BUCKET = "fxa-prod-basket"
+GCS_PREFIX = "last-active-timestamp"
+
+with models.DAG(
+    "adjust_import", default_args=default_args, schedule_interval="15 1 * * *"
+):
+    # Create the table with yesterday's data
+    gcp_conn_id = "google_cloud_derived_datasets"
```

Comment:
Any reason not to use shared-prod?

Reply:
Will change. I think I only used this because the GKE cluster is in derived-datasets, but that's not relevant to this DAG.

Reply:
Reverted this change back to `google_cloud_derived_datasets`. That is the name of the relevant connection in Airflow. There is no connection named "shared_prod".

Reply:
Right, of course. Can we write to shared prod with that connection ID?

Reply:
> Right, of course. Can we write to shared prod with that connection ID?

Yes.